master/pkg/model/experiment_config.go

```diff
@@ -80,18 +81,24 @@ func (d *DeviceConfig) UnmarshalJSON(data []byte) error {
 	return errors.Wrap(json.Unmarshal(data, DefaultParser(d)), "failed to parse device")
 }
 
+// ResourcePoolName includes the (optional) ResourceManager name & ResourcePool name.
+type ResourcePoolName struct {
+	ResourceManager null.String
```

Comment:
does ```ResourceManager``` need to be ```null.String``` or can it just be a ```string```? 

```ResourcePool``` is "optional" in the sense that we will do some defaulting logic if the user does not specify it

Can we take the same approach where we default and fill in ```ResourceManager``` always?

Reply:
outdated